cmd/rootMain.go

```diff
@@ -117,6 +117,15 @@ func rootMain(cmd *cobra.Command, args []string) exitCode {
 		return exitCode{1}
 	}
 
+	// Mountpath is not cleaned up correctly if removed as part of deleting rundir, so it's placed
+	// in a separate location.
+	mountpath, err := ioutil.TempDir(cachedir, "mnt")
```

Comment:
Hmm, why'd you decide to leave https://github.com/puppetlabs/wash/blob/master/fuse/core.go#L156 alone (assuming that was the issue)? Just curious.

Reply:
What do you mean "leave it alone"? I made some changes there, but it still didn't work. I couldn't find any leftover goroutines that might be finishing up there, so confused what's going on. May be an actual thread that the fuse library or libfuse is running that I can't find.

Reply:
Sorry, that comment was made before I saw your next commit.